packages/cli/src/telemetry/telemetry-event-relay.service.ts

```diff
@@ -118,6 +122,9 @@ export class TelemetryEventRelay {
 		this.eventService.on('workflow-saved', async (event) => {
 			await this.workflowSaved(event);
 		});
+		this.eventService.on('workflow-post-execute', async (event) => {
+			await this.workflowPostExecute(event);
+		});
```

Comment:
considering how frequently this pattern is repeated in this file, and likely will be repeated in other services as we start using more EventEmitters, I wonder if it makes sense to create a utility function to pass a map of eventName to handlerName, and then set these all up in there instead.